draft-ietf-quic-http.md

```diff
@@ -620,9 +625,8 @@ When a prioritized element is first created, it has a default initial weight of
 orphan placeholder; pushes are dependent on the client request on which the
 PUSH_PROMISE frame was sent.
 
-Requests may override the default initial values by including a PRIORITY frame
-(see {{frame-priority}}) at the beginning of the stream. These priorities
-can be updated by sending a PRIORITY frame on the control stream.
+Requests may override the default initial values by sending a PRIORITY frame
```

Comment:
Including it at stream creation is overriding the default priority.  I think I'd just call this "update" and say that everything has default priority when it's first created.

```suggestion
Priorities can be updated by sending a PRIORITY frame
```
\+ reflow